Public/Src/Cache/MemoizationStore/Library/Tracing/RocksDbMemoizationStoreTracer.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BuildXL.Cache.ContentStore.Interfaces.Logging;
+using BuildXL.Cache.MemoizationStore.Tracing;
+
+namespace BuildXL.Cache.MemoizationStore.Stores
+{
+    internal class RocksDbMemoizationStoreTracer : MemoizationStoreTracer

```

Comment:
>RocksDbMemoizationStoreTracer [](start = 19, length = 29)

Remove this. Moving forward we should not create Tracer types. Especially if they don't add any additional logic.